tests/integration/pilot/minimal/testdata/serving.yaml

```diff
@@ -0,0 +1,1023 @@
+apiVersion: v1
```

Comment:
knative-serving.yaml would be a better name here.

A make target to generate this testdata would be helpful as well. It isn't clear how this testdata was generated. If knative revs, how do we keep track of the new application-specific manifest?

e.g. `make gen-test-app-manifests` which is tied into `make gen`. Perhaps there could be a better target name.

Reply:
I renamed and added a comment where it comes from.

I don't think we want/need/can this to be a part of `gen`, it is manually updated when required